src/App.tsx

```diff
@@ -15,36 +16,116 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+enum SortOption {
+  ALPHABETICALLY = 'alphabetically',
+  BY_LENGTH = 'length',
+  DEFAULT = '',
+}
```

Comment:
[CODE STYLE] - Enum constant names usually start with capital letter. Values can have any case that's necessary.
![image](https://github.com/mate-academy/react_list-of-goods-hooks/assets/63243136/9ccbe635-cf37-409d-958a-a3be6ac5afeb)